sdk/cognitiveservices/cognitiveservices-luis-authoring/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/cognitiveservices-luis-authoring",
   "author": "Microsoft Corporation",
   "description": "LUISAuthoringClient Library with typescript type definitions for node.js and browser.",
-  "version": "3.0.1",
+  "version": "3.1.0-preview",
```

Comment:
@sarangan12 Do you know if the auto publishing process will add the `@next` tag to the released package if the "preview" suffix is present in the version?